package.json

```diff
@@ -48,31 +47,28 @@
     "Formatters",
     "Snippets"
   ],
-  "resolutions": {
-    "minimist": "^1.2.5"
-  },
   "devDependencies": {
     "@types/mocha": "^5.2.6",
-    "@types/node": "^10.14.16",
-    "@types/vscode": "^1.37.0",
+    "@types/node": "^10.17.28",
+    "@types/vscode": "^1.47.0",
```

Comment:
We should probably run `npm audit fix` instead of `npm update --dev` since the latter updates dependencies that don't have security vulnerabilities (according to `npm audit`) like this one.

Reply:
I've tried running just `npm audit fix`, and it seems to break the extension. I'll try some different tactics to avoid updating all the packages.

Reply:
I updated the webpack module (which requires elliptic) and switched an import from require.js to an ES6 import. It now appears to work.